### PR TITLE
Update color set

### PR DESCRIPTION
**Pull request by ricardoom**
added some existent nypl colors to the toolkit.

### CODE REVIEW

File: docs/_sections/color.md [hunk 1]
```diff
@@ -33,22 +33,22 @@ description: Color Usage
 </li><li class="color-box">
   <div class="nypl-dark-gray-swatch">$nypl-dark-gray: darken($nypl-gray, 40%)</div>
 </li><li class="color-box">
-    <div class="nypl-light-gray-swatch">$nypl-light-gray: lighten($nypl-gray, 40%)</div>
+    <div class="nypl-light-gray-swatch reverse">$nypl-light-gray: lighten($nypl-gray, 40%)</div>
```
Comment (wlla) [inline docs/_sections/color.md]: Add hex value to text label

File: docs/_sections/color.md [hunk 2]
```diff
@@ -33,22 +33,22 @@ description: Color Usage
...
   <div class="nypl-black-swatch">$nypl-black: #111</div>
 </li>
 </ul>
 <ul class="swatches reverse">
 <li class="color-box">
-  <div class="nypl-yellow-swatch main">$nypl-yellow: #fee24a</div>
+  <div class="nypl-yellow-swatch main reverse">$nypl-yellow: #fee24a</div>
 </li><li class="color-box">
-  <div class="nypl-yellow-tint-swatch">$nypl-yellow-tint: tint($nypl-yellow, 50%)</div>
+  <div class="nypl-yellow-tint-swatch reverse">$nypl-yellow-tint: tint($nypl-yellow, 50%)</div>
```
Comment (wlla) [inline docs/_sections/color.md]: Add hex value to text label

File: docs/_sections/color.md [hunk 3]
```diff
@@ -33,22 +33,22 @@ description: Color Usage
...
 </li><li class="color-box">  
-  <div class="nypl-orange-swatch">$nypl-orange: #ffb81d</div>
+  <div class="nypl-orange-swatch reverse">$nypl-orange: #ffb81d</div>
 </li><li class="color-box">  
-  <div class="nypl-orange-desaturated-swatch">$nypl-orange-desaturated:<br>desaturate($nypl-orange, 40%)</div>
+  <div class="nypl-orange-desaturated-swatch reverse">$nypl-orange-desaturated:<br>desaturate($nypl-orange, 40%)</div>
```
Comment (wlla) [inline docs/_sections/color.md]: Add hex value to text label

File: docs/_sections/color.md [hunk 4]
```diff
@@ -57,6 +57,24 @@ description: Color Usage
...
+  <div class="nypl-green-tint-swatch reverse">$nypl-green-tint:<br>tint($nypl-green, 40%)</div>
+</li>
+</ul>
+<ul class="swatches">
+<li class="color-box">
+  <div class="nypl-teal-swatch main">$nypl-teal: #07818d</div>
+</li><li class="color-box">
+  <div class="nypl-teal-dark-swatch">$nypl-teal-dark: #047074</div>
+</li><li class="color-box">  
+  <div class="nypl-teal-tint-swatch reverse">$nypl-teal-tint:<br>tint($nypl-teal, 50%)</div>
```
Comment (wlla) [inline docs/_sections/color.md]: Add hex value to text label

File: docs/_sections/color.md [hunk 5]
```diff
@@ -57,6 +57,24 @@ description: Color Usage
 </li><li class="color-box">
   <div class="nypl-green-dark-swatch">$nypl-green-dark: #497629</div>
 </li><li class="color-box">  
-  <div class="nypl-green-tint-swatch">$nypl-green-tint:<br>tint($nypl-green, 40%)</div>
+  <div class="nypl-green-tint-swatch reverse">$nypl-green-tint:<br>tint($nypl-green, 40%)</div>
```
Comment (wlla) [inline docs/_sections/color.md]: Add hex value to text label

File: docs/_sections/color.md [hunk 6]
```diff
@@ -57,6 +57,24 @@ description: Color Usage
...
+  <div class="nypl-teal-tint-swatch reverse">$nypl-teal-tint:<br>tint($nypl-teal, 50%)</div>
+</li>
+</ul>
+<ul class="swatches">
+<li class="color-box">
+  <div class="nypl-purple-swatch main">$nypl-purple: #72256d</div>
+</li><li class="color-box">
+  <div class="nypl-purple-dark-swatch">$nypl-purple-dark: #52174f</div>
+</li><li class="color-box">  
+  <div class="nypl-purple-tint-swatch reverse">$nypl-purple-tint:<br>tint($nypl-purple, 40%)</div>
```
Comment (wlla) [inline docs/_sections/color.md]: Add hex value to text label

**Comment by ricardoom** [comment]
@wlla gotcha. will make some annotations and add the calculated #hheexx value

**Pull request closed without merging**